### ISSUE TITLE
Make cli run under pyodide-cli?

### ISSUE THREAD

**Issue by joemarshall** [pyodide/pyodide-lock#19]
It seems weird to have the CLI for this run with `pyodide-lock` not as a subcommand of `pyodide`, e.g. `pyodide lock` or `pyodide lockfile`? pyodide-lock has minimal dependencies (typer and rich)?

**Comment by rth** [comment]
Yes, absolutely it's planned. Mostly stuck at the API design step so far #4
